plugins/reveal.js

```diff
@@ -82,7 +82,8 @@ class Reveal {
   }
 
   nextSlide() {
-    return this.page.evaluate(_ => Reveal.next());
+    this.page.evaluate(_ => Reveal.next());
+    return this.page.evaluate(_ => Reveal.layout());
```

Comment:
I think this should be:

```suggestion
    return this.page.evaluate(_ => Reveal.next()).then(this.page.evaluate(_ => Reveal.layout()));
```

WDYT?